Dockerfile

```diff
@@ -12,7 +12,8 @@ RUN wget https://apt.llvm.org/llvm-snapshot.gpg.key && \
         gcc-multilib \
         lld-12 \
         lldb-12 \
-        meson && \
+        meson \
+        sudo && \
```

Comment:
What do you need it for?

Reply:
```[0/3] Generating btf with a custom command.
+ OUTPUT=../src/bpf/vmlinux.h
+ sudo bpftool btf dump file /sys/kernel/btf/vmlinux format c
Error: failed to load BTF from /sys/kernel/btf/vmlinux: No such file or directory
```

Reply:
```
Configuring lockcd.service using configuration
meson.build:48: WARNING: Passed invalid keyword argument "depends".
WARNING: This will become a hard error in the future.
Configuring config.toml using configuration
meson.build:67: WARNING: Passed invalid keyword argument "depends".
WARNING: This will become a hard error in the future.
Configuring daemon.json using configuration
Build targets in project: 3
Found ninja-1.8.2 at /usr/bin/ninja
ninja: Entering directory `.'
[0/3] Generating btf with a custom command.
+ OUTPUT=../src/bpf/vmlinux.h
+ sudo bpftool btf dump file /sys/kernel/btf/vmlinux format c
Error: failed to load BTF from /sys/kernel/btf/vmlinux: No such file or directory
```

Reply:
Ah, true, you are right. Let's keep it then.